projects/playground/src/app/lazy-module/lazy-module.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { LazyModuleComponent } from './lazy-module.component';
```

Comment:
probably we don't want this file either. (For the other 2 introduced components we don't have any)

Reply:
Let's have a chat about how we unify all these components, because we do a mixture (across the protected and unprotected components as well).